scripts/microbe_stage/organelle_table.as

```diff
@@ -512,7 +512,7 @@ void setupOrganelles(){
 
     respiratoryProtien.mass = 0.1;
     respiratoryProtien.gene = "m";
-    respiratoryProtien.mesh = "mitochondrion.mesh";
```

Comment:
The typo "Protien" would be awesome to be finally fixed...

Reply:
So should I fix that then merge it?

Reply:
I would appreciate having this typo finally fixed. What are you referring to in your comment about mergin *it*? I don't think this pull request should be merged before the emerging mess of each model having separate rotation in the code is fixed.

Reply:
I mean it’s only the chemoplast with the messed up rotation

Reply:
I asked them to fix it but they haven’t yet

Reply:
I corrected the protein typo